ozpricechecker/pricefinderapp/tests/models/test_product.py

```diff
@@ -18,6 +20,20 @@ def test_str_no_name(self):
     def test_str_name(self):
         product = Product.objects.create(store=self.store, prod_url='/1234/Tomatoes', name='Super Tomatoes')
         self.assertEqual(str(product), 'Woolworths - Super Tomatoes')
+    
+    def test_latest_prod_price_date(self):
+        product = Product.objects.create(store=self.store, prod_url='/1234/Tomatoes', name='Super Tomatoes')
```

Comment:
Can we add a test for accessing the Properties when ProductPrice table is empty

Reply:
Add now if an exception caused otherwise later

Reply:
Fixed exception.